src/include/denc.h

```diff
@@ -51,6 +51,7 @@
 
 #include "common/convenience.h"
 #include "common/error_code.h"
+#include "ceph_ver.h"
```

Comment:
This has been discussed in the last core syncup. `denc.h` is included almost everywhere, and thus `ceph_ver.h` will be too. The problem is we need to rebuild far, far more when it changes. @aclamk is addressing the issue.